schemas/fingerprint-server-api.yaml

```diff
@@ -19,20 +23,33 @@ servers:
   - url: https://ap.api.fpjs.io
     description: Asia (Mumbai)
 security:
-  - ApiKeyHeader: [ ]
-  - ApiKeyQuery: [ ]
+  - ApiKeyHeader: []
+  - ApiKeyQuery: []
+x-readme: 
+  samples-languages: 
+  - curl
+  - node
+  - php
+  - python
+  - csharp
+  - java
+  - go
 paths:
   /events/{request_id}:
     get:
       tags:
-        - Fingerprint
```

Comment:
It will cause breaking changes to our Server APIs. Tag is responsible to group methods in API class.
Now `swagger-codegen` generates one class `FingerprintApi`, after applying changes we will get `EventsApi` and `VisitorsApi`.

Reply:
Unfortunate, okay, I will reverse the change and try to figure out something else.

Reply:
The next best thing is using a transformer to just remove all tags for the Readme version: https://github.com/fingerprintjs/fingerprint-pro-server-api-openapi/pull/48/commits/31927c1bc115e95602b5a9abcbed5c2a019e12c9

Not sure I really understand the swagger-codegen limitation, though. Surely it's super common to group methods by an entity using tags the way I have done and still having all methods on a single SDK class. Like right now the SDKs presumably do some kind of mapping like this: 
```
client.getEvent = FingerprintApi.getEvent
client.getVisitorHistory = FingerprintApi.getVisitor
```

Couldn't we just change that mapping to 
```
client.getEvent = EventsApi.getEvent
client.getVisitorHistory = VisitorsApi.getVisitor
```
?

I am not suggesting we do all that just to make the docs look slightly prettier, just seems like it should be possible. Have a look at the last commit, that should be good enough from my POV, thanks :)

Reply:
Resolved by using `Events` and `Visitors` tags in the source schema and replacing them with `Fingerprint` for the default build schema.